addons/nodelocaldns/nodelocaldns.yaml

```diff
@@ -134,9 +134,10 @@ spec:
           requests:
             cpu: 25m
             memory: 5Mi
-        args: [ "-localip", "__PILLAR__LOCAL__DNS__,__PILLAR__DNS__SERVER__", "-conf", "/etc/Corefile", "-upstreamsvc", "kube-dns-upstream" ]
```

Comment:
This seems redundant because we already provide all those flags as part of the command on L132, so I removed this line.